### PR TITLE
How to deal with permission problems when running docker on Ubuntu/WSL

### PR DESCRIPTION
**Pull request by ericpromislow**
Works for other linux distros as well.

Addresses https://github.com/rancher-sandbox/rancher-desktop/issues/1156

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -88,3 +88,15 @@ kubectl delete node lima rancher-desktop
 
 **A:** No, but there are plans to add compatibility.
 
+<!-- #1156 -->
+**Q: How do I fix `permission denied` errors when trying to use Docker no WSL? **
```
Comment (gunamata) [inline docs/faq.md:92]: Did you mean `Docker on WSL`?

Comment (ericpromislow) [reply docs/faq.md]: yse :)

**Review by gunamata — APPROVED** [review]

File: docs/faq.md (hunk 1 above)
Comment (zhaoyuedas) [reply docs/faq.md]: I got the same problem on wls2, the solution works perfectly，thanks a lot!

Comment (zhaoyuedas) [reply docs/faq.md]: > 

I got the same problem on wls2, the solution works perfectly，thanks a lot!

**Comment by zhaoyuedas** [comment]
> Works for other linux distros as well.
> 
> Addresses [rancher-sandbox/rancher-desktop#1156](https://github.com/rancher-sandbox/rancher-desktop/issues/1156)

I got the same problem on wls2, the solution works perfectly，thanks a lot!

**Pull request merged**